docs/API.TraversingOutboundAndInboundLinksWithXgAndYg.md

```diff
@@ -287,9 +287,9 @@ This query retrieves all infotons that point to the matched infotons through the
 
 If you want to expand along several paths using a single query, you can add several expansion expressions separated by the pipe ('|') operator.
 
-For example, the following query searches for organizations whose name contains "Marriott", and also retrieves Quote and Instrument infotons that are issues by those organizations. Note the use of the '|' operator to expand along two different inbound links.
+For example, in a single query the following call retrieves both the place that John Smith was born (via an outbound link), and all the places where John Smith lived (via a combination of inbound and outbound links):
 
-    <cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott&yg=<isQuoteOf.mdaas|<isIssuedBy.mdaas
+    <cm-well-host>/example.org/Individuals/John_Smith?yg=<hasTenant.xmpl>location.locn|>birthPlace.madsrdf
```

Comment:
We probably want to reference a RDF sample here...
The query alone has no context. You can modify the sample I gisted, or use something of your own...
Doesn't really matter, but data sample as context is needed here IMO.